src/structures/message.ts

```diff
@@ -63,8 +62,10 @@ export class Message extends Base {
     this.guildID = data.guild_id
```

Comment:
What about making `*_id` to Snowflake

Reply:
I'm not sure 🤔

Reply:
It's just my think and we can change later so I think I can mark this comment resolved